packaging/ansible-runner-service-project/project/roles/gluster-brick-create/tasks/main.yml

```diff
@@ -71,15 +73,18 @@
 
 # rc 1 = Filesystem already exists
 - name: Create an xfs filesystem
-  command: "mkfs.xfs -f -K -i size=512 -n size=8192 {% if 'raid' in disktype %} -d sw={{ diskcount }},su={{ stripesize }}k {% endif %} /dev/{{ vgname }}/{{ lvname }}"
+  ansible.builtin.command: >
+    "mkfs.xfs -f -K -i size=512 -n size=8192 {% if 'raid' in disktype %} -d sw={{ diskcount }},su={{ stripesize }}k
+    {% endif %} /dev/{{ vgname }}/{{ lvname }}"
   register: resp
   failed_when: resp.rc not in [0, 1]
   changed_when: resp.rc == 0
 
 - name: Create the backend directory, skips if present
-  file:
+  ansible.builtin.file:
     path: "{{ mntpath }}"
     state: directory
+    mode: '755'
```

Comment:
To specify the mode, `'755'` notation is used in some places and `0755` in other places.  Since it's part of cleanup, it would be good to be consistent and preferably use one or the other everywhere.

Reply:
done